applications/vessel-history/src/features/home/Home.tsx

```diff
@@ -69,7 +70,9 @@ const Home: React.FC<LoaderProps> = (): React.ReactElement => {
             vesselID: vessel.id ?? 'NA',
             tmtID: vessel.vesselMatchId ?? 'NA',
           },
-          query: {},
+          query: {
+            aka: aka as any,
```

Comment:
is aka need a string and you know that you have an array, sound better handle the array before assign it to the query